stock_move_line_expiration_date_required/migrations/16.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
+from odoo import SUPERUSER_ID, api, tools
+
+
+def migrate(cr, version):
```

Comment:
@Shide The current version of this module is 16.0.1.0.1 yet. So, you need both to change this directory name and manifest version

Reply:
Done, thanks! :)